js/views/aliases.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * ownCloud - Mail
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Tahaa Karim <tahaalibra@gmail.com>
+ * @copyright Tahaa Karim 2016
+ */
+
+define(function(require) {
+	'use strict';
+
+	var Marionette = require('marionette');
+	var AliasesListView = require('views/aliases-list');
+	var Backbone = require('backbone');
+	var Radio = require('radio');
+
+	return Marionette.CollectionView.extend({
+		collection: null,
+		childView: AliasesListView,
+		currentAccount: null,
+		initialize: function(options) {
+			this.currentAccount = options.currentAccount;
+			this.collection = new Backbone.Collection(this.currentAccount.get('aliases'));
+			this.listenTo(Radio.aliases, 'alias:add', this.addAlias);
+			this.listenTo(Radio.aliases, 'alias:remove', this.removeAlias);
+		},
+		addAlias: function(alias) {
+			this.collection.add(alias);
```

Comment:
so you expect a single alias here, but you actually pass an array of aliases when triggering the event. Hence, the view is rendered incorrectly then.